enterprise/cypher/slotted-runtime/src/main/scala/org/neo4j/cypher/internal/runtime/slotted/SlottedPipeBuilder.scala

```diff
@@ -299,52 +274,12 @@ class SlottedPipeBuilder(fallback: PipeBuilder,
 
       case _ =>
         fallback.build(plan, source)
-        //throw new CantCompileQueryException(s"Unsupported logical plan operator: $plan")
+      //throw new CantCompileQueryException(s"Unsupported logical plan operator: $plan")
     }
     pipe.setExecutionContextFactory(SlottedExecutionContextFactory(slots))
     pipe
   }
 
-  private def refSlotAndNotAlias(slots: SlotConfiguration, k: String) = {
-    !slots.isAlias(k) &&
-      slots.get(k).forall(_.isInstanceOf[RefSlot])
-  }
-
-  private def translateColumnOrder(slots: SlotConfiguration, s: plans.ColumnOrder): pipes.ColumnOrder = s match {
-    case plans.Ascending(name) =>
-      slots.get(name) match {
-        case Some(slot) => pipes.Ascending(slot)
-        case None => throw new InternalException(s"Did not find `$name` in the slot configuration")
-      }
-
-    case plans.Descending(name) =>
-      slots.get(name) match {
-        case Some(slot) => pipes.Descending(slot)
-        case None => throw new InternalException(s"Did not find `$name` in the slot configuration")
-      }
-  }
-
-  private def createProjectionsForResult(columns: Seq[String], slots: SlotConfiguration) = {
-    val runtimeColumns: Seq[(String, commandExpressions.Expression)] =
-      columns.map(createProjectionForIdentifier(slots))
-    runtimeColumns
-  }
-
-  private def createProjectionForIdentifier(slots: SlotConfiguration)(identifier: String) = {
-    val slot = slots.get(identifier).getOrElse(
-      throw new InternalException(s"Did not find `$identifier` in the slot configuration")
-    )
-    identifier -> SlottedPipeBuilder.projectSlotExpression(slot)
-  }
-
-  private def buildPredicate(expr: frontEndAst.Expression)
-                            (implicit context: PipeExecutionBuilderContext, planContext: PlanContext): Predicate = {
-    val rewrittenExpr: frontEndAst.Expression = rewriteAstExpression(expr)
-
-    expressionConverters.toCommandPredicate(rewrittenExpr).rewrite(KeyTokenResolver.resolveExpressions(_, planContext))
-      .asInstanceOf[Predicate]
-  }
-
```

Comment:
Why do we need to move this?

Reply:
I moved all private methods to the bottom.